crates/bevy_winit/src/winit_windows.rs

```diff
@@ -59,6 +61,30 @@ impl WinitWindows {
             .with_decorations(window_descriptor.decorations),
         };
 
+        let WindowResizeConstraints {
+            min_width,
+            min_height,
+            max_width,
+            max_height,
+        } = check_resize_constraints(window_descriptor.resize_constraints.clone());
+        let min_inner_size = LogicalSize {
+            width: min_width,
+            height: min_height,
+        };
+        let max_inner_size = LogicalSize {
+            width: max_width,
+            height: max_height,
+        };
+
+        #[allow(unused_mut)]
+        let mut winit_window_builder = if max_width.is_finite() && max_height.is_finite() {
```

Comment:
In this case I don't think the `mut` will ever actually be used at all, since the builder is always consumed in the next clause
```suggestion
        let winit_window_builder = if max_width.is_finite() && max_height.is_finite() {
```